plugins/inventory/proxmox.py

```diff
@@ -308,12 +308,15 @@ def _get_auth(self):
 
     def _get_json(self, url, ignore_errors=None):
 
-        if not self.use_cache or url not in self._cache.get(self.cache_key, {}):
+        data = []
 
-            if self.cache_key not in self._cache:
-                self._cache[self.cache_key] = {'url': ''}
+        if self.use_cache and not self.update_cache:
```

Comment:
Even if `self.update_cache` is set, this specific query might already be cached.
```suggestion
        if self.use_cache:
```

Reply:
I was thinking about this as well. Background is the question whether a single entry could be missing. From debugging I observed that either the self._cache[self.cache_key] has content or not. Anyway, it should not harm.